libraries/chain/include/scorum/chain/database/database.hpp

```diff
@@ -49,16 +49,27 @@ class database : public chainbase::database,
 {
 
 public:
-    database();
+    enum creation_option
+    {
+        opt_none = 0,
+        opt_log_hardforks = 1 << 0,
```

Comment:
Overhead. We all know degrees of two (1, 2).
Replace opt_log_hardforks by 1 and opt_notify_virtual_op_applying by 2